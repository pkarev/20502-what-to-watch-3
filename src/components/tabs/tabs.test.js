import React from 'react';
import renderer from 'react-test-renderer';
import Tabs from './tabs';
import Tab from './tab';

const tabs = [
  {
    name: `firstTab`,
  },
  {
    name: `secondTab`,
  }
];

it(`Render Component`, () => {
  const tree = renderer
  .create(
      <Tabs activeTab={`${tabs[0].name}`}>
        {tabs.map((tab) => (
          <Tab name={tab.name} key={tab.name}>
            <div>
              {tab.name}
            </div>
          </Tab>
        ))}
      </Tabs>
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
