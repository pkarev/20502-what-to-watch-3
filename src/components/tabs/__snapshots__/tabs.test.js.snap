// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Component 1`] = `
Array [
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item movie-nav__item--active"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          firstTab
        </a>
      </li>
      <li
        className="movie-nav__item"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          secondTab
        </a>
      </li>
    </ul>
  </nav>,
  <div>
    firstTab
  </div>,
]
`;
